### PR TITLE
Basic example

### PR DESCRIPTION
**Pull request by sblack4**
>

### CODE REVIEW

**Comment by n01d** [comment]
Thank you for your contributions. I reached out internally again to find a maintainer for the project. Thank you for your patience.

File: examples/basic/main.tf [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+# main
+# =============================================
+
+module "aks" {
+  source = "../../"
+  #   source = "github.com/Azure/terraform-azurerm-aks?ref=v2.0"
+
+  prefix             = var.name
+  kubernetes_version = local.aks_version
+  CLIENT_ID          = azuread_application.aks_service_principle.application_id
```
Comment (yupwei68) [inline examples/basic/main.tf]: We could use TF_VAR to set environment variable TF_VAR_client_id to simplify the test code.

File: variables.tf [hunk 2]
```diff
@@ -50,3 +50,28 @@ variable "public_ssh_key" {
...
+
+variable "network_profile" {
+  description = "Variables defining the AKS network profile config"
+  type = object({
+    network_plugin     = string
+    network_policy     = string
+    dns_service_ip     = string
+    docker_bridge_cidr = string
+    pod_cidr           = string
+    service_cidr       = string
```
Comment (yupwei68) [inline variables.tf]: Could we create these resources in this module instead of importing them as variables?

**Comment by yupwei68** [comment]
Hi @sblack4  , thanks for opening this PR. Sorry that I have to close it now.

**Pull request closed without merging**